src/App.js

```diff
@@ -18,6 +21,14 @@ function App() {
     fetchData();
   }, []);
 
+  const searchHandler = async () => {
+    if (!zip) {
+      setError('please add zip!');
+    } else {
+      const data = await fetchBusinesses(zip, search);
+      return setBusinesses(data);
+    }
```

Comment:
Don't forget to reset the error state in the else block.

Reply:
> Don't forget to reset the error state in the else block.

Thanks Tanner, I have added this.